libs/wingc/src/lsp/completions.rs

```diff
@@ -585,22 +565,19 @@ fn get_current_scope_completions(
 				return vec![];
 			}
 
-			"json_map_literal" => {
-				return vec![];
-			}
-
 			"custom_type" => in_type = true,
 			_ => {}
 		}
 	}
 
 	let found_stmt_index = scope_visitor.found_stmt_index.unwrap_or_default();
 	let found_env = types.get_scope_env(&scope_visitor.found_scope);
+	let error_buffer = if node_to_complete.has_error() { 1 } else { 0 };
```

Comment:
What do 1 and 0 represent here? (they look sort of like magic numbers to me 🙂)

Reply:
Rewrote this not to need it anymore, should hopefully be more clear